### PR TITLE
HOT-96682: pgbouncer

### PR DESCRIPTION
**Pull request by bgvozdev**

### CODE REVIEW

File: github-for-jira.sd.yml [hunk 1]
```diff
@@ -6,11 +6,25 @@ notifications:
...
+    tag: 1.15.0
+    command: [ "database" ]
+    environment:
+      PGBOUNCER_POOL_MODE: transaction
+      PGBOUNCER_DEFAULT_POOL_SIZE: 15 # Max scale up:
+                                      #   (10 web servers + 6 workers) * 15 pool size  = 240 connections
+                                      # Normal business:
+                                      #   (5 web servers + 3 workers) * 15 pool size = 120 connections
+      PGBOUNCER_SERVER_IDLE_TIMEOUT: 60
+      PGBOUNCER_MAX_CLIENT_CONN: 1000
```
Comment (bgvozdev) [inline github-for-jira.sd.yml:20]: List of available options: https://hello.atlassian.net/wiki/spaces/MICROS/pages/167213858/HOWTO+Run+pgbouncer+as+a+sidecar

Comment (bgvozdev) [reply github-for-jira.sd.yml:20]: https://hello.atlassian.net/wiki/spaces/MICROS/pages/167213858/HOWTO+Run+pgbouncer+as+a+sidecar

File: db/config.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   },
   "production": {
     "dialect": "postgres",
-    "use_env_variable": "PG_DATABASE_URL",
+    "use_env_variable": "DATABASE_URL",
```
Comment (mboudreau) [inline db/config.json:12]: Nice! Could you maybe increased the max pool count below to 15 then?

File: github-for-jira.sd.yml (hunk 1 above)
Comment (bgvozdev) [reply github-for-jira.sd.yml:20]: https://sourcegraph-frontend.internal.shared-prod.us-west-2.kitt-inf.net/bitbucket.org/atlassianlabs/prlinks/-/blob/evzijst-prlinks.sd.yml sample prod config

File: github-for-jira.sd.yml [hunk 3]
```diff
@@ -296,14 +311,14 @@ environmentOverrides:
...
-      max: 10
+      max: 10 # keep in sync with PGBOUNCER_DEFAULT_POOL_SIZE
       metrics: *CpuMemAlbScalingRules
     workers:
       - name: Worker
         scaling:
           instance: c5.2xlarge
           min: 3
-          max: 6
+          max: 6 # keep in sync with PGBOUNCER_DEFAULT_POOL_SIZE
```
Comment (mboudreau) [inline github-for-jira.sd.yml:321]: we could potentially increase this to max 10 if pg bouncer works correctly.

**Review by mboudreau — APPROVED** [review]
Good work :)

**Pull request merged**